### ISSUE TITLE
New to Exporter and Prometheus

### ISSUE THREAD

**Issue by nexusguy59** [znerol/prometheus-pve-exporter#11]
Hello Guys,

        I have installed the exporter on my Prometheus Server however I am getting a http 500 error during the connect process:

127.0.0.1 - - [28/Feb/2018 09:29:14] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:29:29] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:29:44] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:29:59] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:30:14] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:30:29] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -
127.0.0.1 - - [28/Feb/2018 09:30:44] "GET /pve?module=default&target=192.168.1.2 HTTP/1.1" 500 -

This is my pve.yml:
default:
    user: prometheus@pam
    password: sEcr3T!
    verify_ssl: false

Yes this user exists on the promox server and is read only.

Thanks,

**Comment by znerol** [comment]
Can you check whether this is a dupe of #9 by verifying PR #10 

Would be greatly appreciated.

**Comment by nexusguy59** [comment]
It does look like it might be....I am new to this so I have no idea how to fix it.

**Comment by znerol** [comment]
I did merge #10 and released v1.1.1

**Comment by nexusguy59** [comment]
Yes you did i reinstalled and it is working now. Thank you and Good Job!

Michael A Cooper
Linux & VMWare
& Zerto Certified Professional
